resources/src/main.rs

```diff
@@ -55,12 +55,19 @@ async fn intercept_challenge_request(body: &Value) -> String {
         log::info!("{}", challenge_info);
     }
 
-    return challenge.to_string();
+    challenge.to_string()
```

Comment:
This is more idiomatic, so leave it, but it makes me sad sometimes. 😆

Reply:
Same I just recently learned it was the way to do it in rust if it was your return value